src/main/content/_assets/css/guide-card.css

```diff
@@ -42,7 +42,7 @@
 
 .guide_item:hover, .small_guide_item:hover {
     box-shadow:0 2px 4px 0 rgba(63,70,89,0.59);
-    border-left: solid 0.3em #C43804;
+    border-left: solid 9px #C43804;
```

Comment:
I would use `border-color: #C43804;` instead, this way if the border does change in .guide_item or small_guide_item we don't run into this snag again.

Reply:
I plugged that in and it looks good.  I'll make the change. Thanks.